jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/javacore/parser/j9/section/platform/PlatformPatternMatchers.java

```diff
@@ -43,8 +43,7 @@ public class PlatformPatternMatchers {
 	public static final Matcher Linux = CommonPatternMatchers.generateMatcher("Linux", Pattern.CASE_INSENSITIVE);
 	public static final Matcher AIX = CommonPatternMatchers.generateMatcher("AIX", Pattern.CASE_INSENSITIVE);
 	public static final Matcher z_OS = CommonPatternMatchers.generateMatcher("z/OS", Pattern.CASE_INSENSITIVE);
-	public static final Matcher J9Signal_1 = CommonPatternMatchers.generateMatcher("J9Generic_Signal:", Pattern.CASE_INSENSITIVE);
-	public static final Matcher J9Signal_2 = CommonPatternMatchers.generateMatcher("J9Generic_Signal_Number:", Pattern.CASE_INSENSITIVE);
```

Comment:
We shouldn't be removing recognition of these.

Reply:
I can put them back but FYI, these were only used in one place in the codebase and that use is being replaced with a check for `Signal_Number` instead, so these will be dead code.

Reply:
Ok, I think I now see what you're doing: you're changing the rule which previously looked for `J9Generic_Signal:` or `J9Generic_Signal_Number:` and are instead looking for `Signal_Number:` so we will no longer need `resolveGenericSignal()`. I'm fine if you want to remove the patterns `J9Signal_[12]`.